tools/install.sh

```diff
@@ -161,41 +161,25 @@ debian=("Debian" "Ubuntu")
 redhat=("Red" "CentOS" "Fedora" "Oracle") 
 
 if [[ " ${debian[*]} " =~ " $os_distro " ]]; then
-    mkdir -p ../modules/document_repository/user_uploads
-    mkdir -p ../modules/data_release/user_uploads
-    sudo chown www-data.www-data ../modules/document_repository/user_uploads
-    sudo chown www-data.www-data ../modules/data_release/user_uploads
-    sudo chown www-data.www-data ../smarty/templates_c
+    mkdir -p -m 770 ../modules/document_repository/user_uploads
+    mkdir -p -m 770 ../modules/data_release/user_uploads
     # Make Apache the group for project directory, so that the web based install
     # can write the config.xml file.
-    sudo chgrp www-data ../project
-    sudo chmod 770 ../project
+    chmod 770 ../project
 elif [[ " ${redhat[*]} " =~ " $os_distro " ]]; then
-    mkdir -p ../modules/document_repository/user_uploads
-    mkdir -p ../modules/data_release/user_uploads
+    mkdir -p -m 770 ../modules/document_repository/user_uploads
+    mkdir -p -m 770 ../modules/data_release/user_uploads
```

Comment:
This doesn't seem to do what the PR description says (not this line, the lines below it where things are still being chown'd to apache.apache)

Reply:
You're right. I'll remove them

Reply:
Even if `group` would have `RWX` on the directories, files created by Apache would have ownership of www-data:www-data and Lorisadmin would not be able to modify/erase them and possibly read them.
If you want that, it would require to set